transport-api/src/main/java/io/scalecube/cluster/transport/ServiceLoaderUtil.java

```diff
@@ -0,0 +1,37 @@
+package io.scalecube.cluster.transport;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.ServiceLoader;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+public final class ServiceLoaderUtil {
```

Comment:
i think we should coniser this utility as part of scalecube utils and not as part of transport
as it appears to be common across projects.

Reply:
i saw you added it to app utils and i was thinking app utils is more about packaging
maybe we just need new module scalecube-utils....